slave/buildslave/test/unit/test_commands_utils.py

```diff
@@ -108,7 +109,7 @@ def tearDown(self):
             if os.path.exists(self.target):
                 shutil.rmtree(self.target)
         except Exception:
-            print "\n(target directory was not removed by test, and cleanup failed too)\n"
+            print("\n(target directory was not removed by test, and cleanup failed too)\n")
```

Comment:
The prints are still in the tests, because I'm assuming they are there to tell the tester if something goes wrong. I don't see any benefit to making it go into a log